net/ampr-ripd/files/ampr-ripd-init

```diff
@@ -17,14 +17,18 @@ start() {
 
 		exit 1
 	fi
+	config_load ampr-ripd
+	config_get enabled network enabled
```

Comment:
You can also specify a default value like so in case it doesn't exist:
```suggestion
	config_get enabled network enabled 1
```
Same goes for the other values if they have a default.